src/main/java/gov/loc/repository/bagit/hash/StandardSupportedAlgorithms.java

```diff
@@ -6,9 +6,9 @@
 public enum StandardSupportedAlgorithms implements SupportedAlgorithm{
   MD5("MD5"),
   SHA1("SHA-1"),
```

Comment:
why is this `"SHA-1"` now different than the others? Now the others don't have this _dash_ anymore, but this one still has it.

Reply:
Good point. I will update to include sha1 as well as tests on very basic valid test bags